packages/core/admin/admin/src/content-manager/hooks/useDragAndDrop.js

```diff
@@ -9,11 +9,12 @@ import { useKeyboardDragAndDrop } from './useKeyboardDragAndDrop';
  *
  * @type {{
  *  type?: string,
- *  index: number,
+ *  index: number | Array<number>,
```

Comment:
To simplify the code, could we always make it an array of numbers? For simple cases it would be an array with just 1 number inside. You'd have to update the other places where it's called of course

Reply:
I'd challenge that to ask, why does this hook have to know about node paths which are a unique problem to the scope of Slate/Blocks?

Is there a reason we must use path arrays instead or using a single value? 🤔

I'm not sure about splitting this API in that way – interested to learn more!

Reply:
I think what makes blocks special is that you can drag list items out of lists. So you have a nested structure, not everything is on the same level

Reply:
I did try with a single value initially and struggled to move list-items as we need to unique ids and map it back to move blocks using path, and with nested lists its going to be a lot problematic.